src/main/groovy/com/vanniktech/android/junit/jacoco/GenerationPlugin.groovy

```diff
@@ -301,6 +301,19 @@ class GenerationPlugin implements Plugin<Project> {
     }
 
     private static boolean shouldIgnore(final Project project, final JunitJacocoExtension extension) {
-        return extension.ignoreProjects?.contains(project.name)
+        boolean result = extension.ignoreProjects?.contains(project.name) || extension.ignoreProjects?.contains(project.path)
+        if (result) {
+            // regex could be slower
+            return true
+        }
+
+        extension.ignoreProjects?.each {
+            if (project.name.find(it) || project.path.find(it)) {
+                result = true
```

Comment:
we don't need to set this anymore do we? or is the return statement below for the each function?

Reply:
It's necessary because Groovy's for each is weird :/ I changed it to a for-each loop.